mlir/utils/performance/perfRunner.py

```diff
@@ -1447,8 +1470,32 @@ def main(args=None):
          help='Force a set of datatypes'
     )
 
+    parser.add_argument("--rv", "--rocprof-version", choices=['1', '2'],
+        default='2',
+        help="Use rocprof (V1) or rocprofv2.")
+
     parsed_args = parser.parse_args(args)
 
+    global getNanoSeconds, getBankConflict, ROCPROF, ROCPROF_OPTS
```

Comment:
Swap functions, options, and filenames based on the option.

Reply:
... why are we keeping rocprof v1 support?